### ISSUE TITLE
Notifications when benchmark_runs complete

### ISSUE THREAD

**Issue by tgxworld** [ruby-bench/ruby-bench-web#22]
Currently we're using [Taskspooler](http://viric.name/soft/ts/) to run our benchmarks. The log is saved in the server's tmp folder but it'll be good if we can log it somewhere.

**Comment by tgxworld** [comment]
Also, we wouldn't know if the job fails.

**Comment by farukaydin** [comment]
@tgxworld we can generate notifications without looking benchmark run is complete or not. We just need after create callback that compare the benchmark run result with the one before. If there is material change we can send mail to all specific benchmark subscribers.

Maybe we move from delayed job to sidekiq or resque. While we moving, we can analyze job is done, or fail or logged etc. Admin UI also see which job is running and where.(Also sam request it)

@SamSaffron I wonder what is your opinions for both of the topic?

**Comment by farukaydin** [comment]
Bytheway if the structure is okey for you, I'd like to work on it.

**Comment by tgxworld** [comment]
@farukaydin Sorry that this issue might not be too clear. I created this issue initially because we were running the benchmark runs through a heroku worker which would SSH into our dedicated server and run the scripts. However, this prevented us from deploying frequently to heroku as heroku restarts the worker. Therefore, I resorted to taskspooler which basically sends the command to a queue on our dedicated server. However, I have to manually check the logs now. In summary, this issue is meant to address the problem where we're not logging the benchmarking jobs instead of the comparing benchmark results.

I'm wanted to move away from delayed_job, perhaps you might be interested in throwing up a PR for it?

Let me know if you've got more questions! :smile:
